app/models/local_restriction.rb

```diff
@@ -0,0 +1,43 @@
+class LocalRestriction
+  FILE_PATH = "lib/local_restrictions/local-restrictions.yaml".freeze
```

Comment:
Perhaps we can wrap the file name in a method, e.g.

```
def file_name
    "lib/local_restrictions/local-restrictions.yaml"
  end
```

And use "mocha" to do the stubbing:

```
before do
    LocalRestriction.any_instance.stubs(:file_name).returns("test/fixtures/local-restrictions.yaml")
end
```

The benefit of using "mocha" is that we don't need to add any other dependencies, and it's a gem that we are already using in other places.

Reply:
If you do decide to go with this approach, there is no point in making any of the methods private.

Reply:
I played around a bit trying to get one of the existing methods stubbed without the need for a new definition. Given the time constraints on this I've stopped trying to get that working 😿